src/views/BookList.vue

```diff
@@ -0,0 +1,50 @@
+<template lang="pug">
+  .book-page-container
+    navBar
+    .books-container
```

Comment:
I think you can add this on home page or do you need in this route? 
If you need I don't like that you add navBar component on each private route, so try to put it only one time for private routes.